### PR TITLE
Avatars and alerts

### PR DESCRIPTION
**Pull request by Naman-Saxena1**

### CODE REVIEW

**Comment by kgajriaAgarwal** [comment]
everything appears good! , btw good design !

File: Components/Alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    width: 100%;
+    box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
+    background-color: #F5F7F9;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-direction: column;
+    flex-wrap: wrap;
+    padding: 1rem;
+    box-sizing: border-box;
```
Comment (Ankur9669) [inline Components/Alert/alert.css:12]: remove box-sizing border box from here and add it to the body or reset css

File: Components/Alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+.alert
+{
+    width: 100%;
+    background-color: var(--primary-dark-blue-color);
+    padding: 0 1rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    flex-grow: 1;
+    box-sizing: border-box;
```
Comment (Ankur9669) [inline Components/Alert/alert.css:25]: remove box-sizing border box from here and add it to the body or reset css

File: Components/Avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    width: 100%;
+    height: 150px;
+    box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
+    background-color: #F5F7F9;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-wrap: wrap;
+    padding: 1rem;
+    box-sizing: border-box;
```
Comment (Ankur9669) [inline Components/Avatar/avatar.css:17]: remove box-sizing border box from here and add it to the body or reset css

File: Components/Avatar/avatar.css [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    justify-content: center;
+    align-items: center;
+    color: var(--background-color);
+    background-color: var(--primary-dark-blue-color);
+    border-radius: 50%;
+}
+
+.avatar-x-large
+{
+    font-size: 4em;
```
Comment (Ankur9669) [inline Components/Avatar/avatar.css:32]: I think here you should use rems instead of em

File: Components/Avatar/avatar.css [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+    width: 6rem;
+    height: 6rem;
+}
+
+.avatar-large
+{
+    font-size: 3em;
```
Comment (Ankur9669) [inline Components/Avatar/avatar.css:39]: I think here you should use rems instead of em

File: Components/Avatar/avatar.css [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+.avatar-img
+{
+    width: 100%;
+    height: 100%;
+    max-width: 100%;
+    max-height: 100%;
+    box-sizing: border-box;
+    border-radius: 50%;
```
Comment (Ankur9669) [inline Components/Avatar/avatar.css:72]: also use object-fit: cover or contain property. This would ensure that your image looks good in avatar.

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
Your naming conventions and code are awesome, but it needs some changes.

File: Components/Alert/alert.css (hunk 1 above)
Comment (Naman-Saxena1) [reply Components/Alert/alert.css]: Done, thanks for this box-sizing advice.

File: Components/Avatar/avatar.css (hunk 4 above)
Comment (Naman-Saxena1) [reply Components/Avatar/avatar.css]: Done, changed to rems

File: Components/Avatar/avatar.css (hunk 6 above)
Comment (Naman-Saxena1) [reply Components/Avatar/avatar.css:71]: Done, thanks, now my avatar image fit properly!

**Pull request merged**